### PR TITLE
changement parmetres pour index

### PR DESCRIPTION
**Pull request by ClementLaussucq**
Guys

1) change forms for registration
2) generate views for devise
3) create 3 fake users : chris, nico, clement
4) coding index (it works)

Check and kiss
Love

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -4,7 +4,6 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
 
-  validates :first_name, :last_name, presence: true, allow_blank: false
-
+  validates :first_name, :last_name, presence: true
```
Comment (nicolasmartin86) [inline app/models/user.rb]: I had removed it from your sublime; I'm not completely sure if you wanted to remove it in the end; sorry @ClementLaussucq  ;-(

File: app/views/pages/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+<div class="container">
+
+  <div class="row">
+    <div class="card-experience">
+      <img src=''>
+    experiences.each do|e| do
+      <div class="experience-description">
+        <p><%= link_to experience.name, experience_path(experience)  %></p>
+        <p> <%= experience.adress %>
```
Comment (nicolasmartin86) [inline app/views/pages/index.html.erb]: typo I think with address

File: app/views/pages/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<div class="banner">
+  <div class="banner-info">
+    <h1>
+      <%= link_to "create a unique experience", <new_experience_path%></new_experience_path%>
```
Comment (nicolasmartin86) [inline app/views/pages/index.html.erb]: typo check you have a "<new_experience_path%></" near the new_experience_path

File: app/views/pages/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+<div class="flexbox">
+  dropbown x2
+</div>
+
+<div class="container">
+
+  <div class="row">
+    <div class="card-experience">
+      <img src=''>
+    experiences.each do|e| do
```
Comment (nicolasmartin86) [inline app/views/pages/index.html.erb]: you're doing an each on "e" columns but you're calling the variable experience, you need to change e to experience

File: app/views/pages/show.html.erb [hunk 5]
```diff
@@ -0,0 +1,17 @@
+ <%= render 'shared/navbar' %>
+
+ <div class="container">
+  <div class="img">
+    IMAGE
+  </div>
+
+  <div class="card-experience">
+    <h1><%= experience.name %></h1>
```
Comment (nicolasmartin86) [inline app/views/pages/show.html.erb]: missing @ no?

File: app/views/pages/index.html.erb (hunk 3 above)
Comment (ClementLaussucq) [reply app/views/pages/index.html.erb]: ok

File: app/views/pages/index.html.erb (hunk 4 above)
Comment (ClementLaussucq) [reply app/views/pages/index.html.erb]: ok

File: app/views/pages/show.html.erb (hunk 5 above)
Comment (ClementLaussucq) [reply app/views/pages/show.html.erb]: ok

**Pull request merged**